src/server/actions/deduplicatedFinding/getDeduplicatedFinding.ts

```diff
@@ -64,7 +64,7 @@ export const getDeduplicatedFindingsAction = async ({
   limit,
   offset = 0,
   sort = {
-    field: MyProjectVulnerabilitiesSorting.FOUND_BY,
+    field: MyProjectVulnerabilitiesSorting.SEVERITY,
```

Comment:
Shouldn't the default sorting be also changed on client?

Reply:
Currently client is using undefined for sort params. Changed the prefetch call in `page` to be undefined as well 👍